KarhooUISDK/Screens/QuoteList/QuoteCell/QuoteView.swift

```diff
@@ -32,33 +32,173 @@ public struct KHQuoteViewID {
+    private lazy var vehicleContainerView = UIView().then {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+    }
+    private lazy var logoLoadingImageView = LoadingImageView().then { logo in
+        logo.accessibilityIdentifier = KHQuoteViewID.logoImage
+        logo.contentMode = .scaleAspectFill
+        logo.layer.masksToBounds = true
+    }
+
+    private lazy var vehicleTypeLabel = UILabel().then {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.accessibilityIdentifier = KHQuoteViewID.name
+        $0.textColor = KarhooUI.colors.text
+        $0.font = KarhooUI.fonts.bodySemibold()
+        $0.numberOfLines = 0
+        $0.lineBreakMode = .byWordWrapping
+    }
+    private lazy var rideDetailStackView = UIStackView().then { rideDetailStackView in
+        rideDetailStackView.translatesAutoresizingMaskIntoConstraints = false
+        rideDetailStackView.accessibilityIdentifier = KHQuoteViewID.rideDetailsStackView
+        rideDetailStackView.axis = .vertical
+        rideDetailStackView.alignment = .leading
+        rideDetailStackView.spacing = UIConstants.Spacing.xSmall
+    }
+    private lazy var vehicleCapacityView = VehicleCapacityView()
+    private lazy var capacityAndPickupTypeContainer = UIStackView().then { stack in
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        stack.accessibilityIdentifier = KHQuoteViewID.capacityStackView
+        stack.axis = .horizontal
+        stack.alignment = .leading
+        stack.spacing = 10.0
+    }
+    private lazy var detailsButton = UIButton().then { button in
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.accessibilityIdentifier = KHQuoteViewID.detailsButton
+        button.backgroundColor = KarhooUI.colors.accent
+        button.setTitleColor(KarhooUI.colors.white, for: .normal)
+        button.titleLabel?.font = KarhooUI.fonts.footnoteRegular()
+        button.setTitle(UITexts.QuoteCell.details, for: .normal)
+        button.clipsToBounds = true
+        button.layer.cornerRadius = UIConstants.CornerRadius.large
+        button.layer.maskedCorners = [.layerMinXMaxYCorner, .layerMaxXMinYCorner]
+        // button temporary hidden, uncomment when Qoute details page will be implemented
+        button.isHidden = true
+    }
+    private lazy var etaStack = UIStackView().then { stack in
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        stack.accessibilityIdentifier = KHQuoteViewID.etaStackView
+        stack.axis = .vertical
+        stack.alignment = .leading
+    }
+    private lazy var etaLabel = UILabel().then { label in
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.accessibilityIdentifier = KHQuoteViewID.eta
+        label.setContentCompressionResistancePriority(.required, for: .horizontal)
+        label.textAlignment = .right
+        label.font = KarhooUI.fonts.headerBold()
+        label.textColor = KarhooUI.colors.black
```

Comment:
shouldn't we use the text color defined in our color palette? black is not that friendly on a white background. 

and there's also dark mode to consider long-term. textColor (or however it's called) would be more descriptive no matter if we're on light or dark mode